### DISCUSSION TITLE
app icon does not show on windows taskbar

### DISCUSSION THREAD

**Discussion by adityalaad** [pear-devs/pear-desktop#272] [Q&A] [answered]
For ease of access, I add the app to my taskbar and it then appears on it and allows me to access the app through the icon. however if I log off and come back to my account, the icon goes to a "blank white page icon". when I click on it, it says the shortcut has been removed or the app has been deleted. but the app is still on my computer and can be opened if I go through my folders.

**Comment by Araxeus** [comment, marked as answer]
Funny timing, I just finished up a feature that will probably fix this problem for you
see #270 (write shortcut to startmenu on launch)

FYI next time you should probably submit an Issue instead of opening a discussion (since this is a bug)

**Reply by adityalaad** [reply]
ah thanks man. 

bit new to GitHub but yeh, next time I'll put any bugs in issue sections.

thanks again

**Reply by Araxeus** [reply]
If it doesn't fix the problem for you, please write here what is the shortcut path that is unresolved
> Shortcut Properties -> Target

(it shouldn't matter if its before/after your icon goes white, since that field isn't supposed to change)

And also write whats the actual path of the application (since you say you can still open it)
